apps/logger_service/app_config.json

```
@@ -0,0 +1,69 @@
+{
+    "include": [
+        "deployment/apps/logger_service/service.json",
+        "deployment/apps/env_service/service.json",
+        "deployment/system_definition/diag/jobs/file_logger/file_logger_did.json"
+    ],
+    "package": "simba.apps",
+    "adaptive_application": {
+        "FileLoggerApp": {
+            "app": {
+                "bootMode": 0,
+                "parms": "-t -10- 100",
```

jak nie uywasz to to
```suggestion
                "parms": "",
```